.github/workflows/rust.yaml

```diff
@@ -35,7 +35,7 @@ jobs:
       - name: cargo tarpaulin xml report
         uses: actions-rs/tarpaulin@v0.1
         with:
-          version: '0.19.1'
```

Comment:
This is completely unrelated haha, this is the `cargo-tarpaulin` version for the GHA flow. I just upgraded to the latest